### DISCUSSION TITLE
Payment not deducted  and subscription is not updated

### DISCUSSION THREAD

**Discussion by mohamedemadkamal1** [community/community#48147] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I am trying to renew my subscription and after submitting the payment it tells me that it is ready to be used but when I open the 
editor (Vscode) I find out that the subscription is not renewed and the money wasn't deducted.

**Comment by Akash1134** [comment]
Hi @mohamedemadkamal1 The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
